src/apps/relay/prom_server.c

```diff
@@ -1,5 +1,6 @@
 #include "prom_server.h"
 #include "mainrelay.h"
+#include "microhttpd.h"
```

Comment:
Already included in .h file - why another include?

Reply:
Good spot. Removed.